rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/530_date_ignore_malformed.yml

```diff
@@ -0,0 +1,115 @@
+---
+"date with ignore_malformed in synthetic source":
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            _source:
+              mode: synthetic
+            properties:
+              date_ignored:
+                type: date
+                ignore_malformed: true
+              date_not_ignored:
+                type: date
+                ignore_malformed: false
+              date_nanos_ignored:
+                type: date_nanos
+                ignore_malformed: true
+              date_nanos_not_ignored:
+                type: date_nanos
+                ignore_malformed: false
+
+  - do:
+      index:
+        refresh: true
+        index: test
+        id: "1"
+        body: { "date_ignored": "2024-02-03T10:12:43.123Z", "date_not_ignored": "2024-02-03T10:12:43.123Z", date_nanos_ignored: "2024-02-03T10:12:43.123456789Z", "date_nanos_not_ignored": "2024-02-03T10:12:43.123456789Z" }
+
+  - do:
+      catch: bad_request
+      index:
+        refresh: true
+        index: test
+        id: "2"
+        body: { "date_ignored": "2024-02-03T10:12:43.123Z", "date_not_ignored": { "key": "a", "value": 10 }, date_nanos_ignored: "2024-02-03T10:12:43.123456789Z", "date_nanos_not_ignored": "2024-02-03T10:12:43.123456789Z" }
+
+  - match: { error.root_cause.0.type: "document_parsing_exception" }
+
+  - do:
+      catch: bad_request
+      index:
+        refresh: true
+        index: test
+        id: "3"
+        body: { "date_ignored": "2024-02-03T10:12:43.123Z", "date_not_ignored": "2024-02-03T10:12:43.123Z", date_nanos_ignored: "2024-02-03T10:12:43.123456789Z", "date_nanos_not_ignored": { "key": "b", "value": 20 } }
+
+  - match: { error.root_cause.0.type: "document_parsing_exception" }
+
+  - do:
+      index:
+        refresh: true
+        index: test
+        id: "1"
```

Comment:
Is it intentional that we use the same id as previous successful request?

Reply:
No this is a mistake.